.github/modify_sitemap.js

```diff
@@ -34,9 +33,71 @@ fs.readFile(inputFile, "utf8", (err, data) => {
   const urlBlockPattern = /(<url>[\s\S]*?<\/url>)/g;
   const xhtmlLinkPattern =
     /(<xhtml:link[^>]*?href="https:\/\/[^\/]+\/[^\/]+\/locations[^"]*"[^>]*?>)/g;
+  const linkPattern =
+    /(<xhtml:link[^>]*?href="https:\/\/[^\/]+\/[^\/]+[^"]*"[^>]*?>)/g;
   const locPattern = new RegExp(`https://${newDomain}\/[^\/]+\/locations`, "i"); // Matches URLs with an extra segment before /locations
   const locDirectPattern = new RegExp(`https://${newDomain}\/locations\/`, "i"); // Matches URLs directly starting with /locations
 
+  const staticDocUrls = [
```

Comment:
Best to dynamically define the urls to make it future proof.

`
const languages = [
  "sw", "bn", "ko", "pl", "tr", "it", "fr", "th", "zh-cn", "si", 
  "vi", "zh-tw", "es", "pt", "ar", "de", "ru"
];

const baseUrl = "https://deriv.com";
const endpoints = [
  "/blog/posts/what-are-interest-rates-and-why-do-they-matter-in-trading",
  "/blog/posts/why-beginners-need-to-have-an-online-trading-demo-account",
  "/blog/posts/the-truth-is-out-5-copy-trading-myths-debunked",
  "/blog/posts/trading-with-crypto-top-3-myths",
  "/blog/posts/eur-usd-holds-strong-before-ecb-meeting",
  "/blog/posts/eu-retail-sales-germany-inflation-rate",
  "/blog/posts/what-are-accumulator-options-a-beginners-guide-to-compounding-growth",
  "/signup"
];

// Generating all the URLs dynamically
const staticDocs = [];

languages.forEach(lang => {
  endpoints.forEach(endpoint => {
    staticDocs.push(`${baseUrl}/${lang}${endpoint}`);
  });
});

// Adding URLs without language prefixes if needed
endpoints.forEach(endpoint => {
  staticDocs.push(`${baseUrl}${endpoint}`);
});
`

Reply:
these are hyper specific scenerios where we donot want to remove for all languages neither for other domains